executor/runtime/docker/docker.go

```diff
@@ -1860,7 +1860,7 @@ func (r *DockerRuntime) setupEFSMounts(parentCtx context.Context, c runtimeTypes
 		mountOptions := append(
 			baseMountOptions,
 			fmt.Sprintf("fsc=%s", c.TaskID()),
-			fmt.Sprintf("source=%s:%s", nfs.Server, nfs.ServerPath),
+			fmt.Sprintf("source=[%s]:%s", nfs.Server, nfs.ServerPath),
```

Comment:
Is putting brackets on the outside here safe to do, safe to do, even if it is a regular hostname or ipv4?

Reply:
Yes, I tested the IPv4 and seemed to work fine. The unit tests tested against the EFS DNS endpoint which also worked.